src/hotspot/share/prims/jvmtiExport.cpp

```diff
@@ -1831,8 +1832,11 @@ void JvmtiExport::post_method_exit(JavaThread* thread, Method* method, frame cur
   HandleMark hm(thread);
   methodHandle mh(thread, method);
 
-  JvmtiThreadState *state = get_jvmti_thread_state(thread);
-
+  JvmtiThreadState *state = nullptr;
+  {
+    ThreadInVMfromJava __tiv(thread);
```

Comment:
Nit: Maybe rename: `__tiv` => `tiv`. The prefix `__` is normally used in macros to avoid potential naming conflicts.

Reply:
Thank you for the update on this!